django/contrib/auth/hashers.py

```diff
@@ -189,9 +196,14 @@ def _load_library(self):
                          self.__class__.__name__)
 
     def salt(self):
-        """Generate a cryptographically secure nonce salt in ASCII."""
-        # 12 returns a 71-bit value, log_2(len(RANDOM_STRING_CHARS)^12) =~ 71 bits
-        return get_random_string(12, RANDOM_STRING_CHARS)
+        """
+        Generate a cryptographically secure nonce salt in ASCII with an entropy
+        of at least `salt_entropy` bits.
+        """
+        # Each character in the salt provides log_2(len(alphabet)) bits of
+        # entropy.
```

Comment:
Really really minor nitpick; if flake8 etc do not complain maybe move `entropy.` up one line; looks really ugly like this especially if the code below is longer :)

Reply:
I cannot move it to the same line per our [coding style](https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/coding-style/#python-style): _"Documentation, comments, and docstrings should be wrapped at 79 characters, even though PEP 8 suggests 72."_. :wink:  but I wrapped it differently to increase readability.